src/style.css

```diff
@@ -1,3 +1,64 @@
+:root {
+  --blue: #00acdc;
+}
+
 body {
+  font-family: Roboto, Helvetica, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You can add a generic family at the end like `serif` or `sans-serif`.